### PR TITLE
Poedit: Add version 2.2.3

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: bucket/poedit.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "2.2.3",
+    "homepage": "https://poedit.net/",
```
Comment (Ash258) [inline bucket/poedit.json]:
```suggestion
    "homepage": "https://poedit.net",
```

File: bucket/poedit.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "description": "Gettext translations editor.",
+    "license": {
+        "identifier": "Shareware,MIT",
+        "url": "https://poedit.net/download"
```
Comment (Ash258) [inline bucket/poedit.json]: Shareware is not mentioned anywhere on this page

Comment (linsui) [reply bucket/poedit.json]: > Up-to-date source code of the Open Source version of Poedit (sans the Pro features included in the above binaries) is available at GitHub under the terms of the MIT license. 

It has a pro edition. Is it a shareware?

Comment (Ash258) [reply bucket/poedit.json]: Where is said you can distribute pro edition?

Comment (linsui) [reply bucket/poedit.json]: The pro edition can be bought and upgrade in app.

Comment (Ash258) [reply bucket/poedit.json]: Then how could it be shareware? 

> Provided free of charge to users, who are allowed and encouraged to make and share copies of the program

There is not mentioned sharing/distributing of program.

Comment (linsui) [reply bucket/poedit.json]: So freeware?

Comment (Ash258) [reply bucket/poedit.json]: https://github.com/vslavik/poedit


> License
> =========
> Poedit is released under the MIT license and you're free to do whatever you want with it and its source code (well, almost :-) -- see the license text).
> See the COPYING file for details on the program's licensing and the icons/README file for details on the icons.
> Windows and macOS versions contain GNU gettext binaries. They are distributed under the GNU General Public License and their source code is available from http://www.gnu.org/software/gettext. If you have difficulties getting them
from there, email me for a copy of the sources.
>>>

**Pull request merged**